ChangeLog.d/drop-msvc-2015-and-armcc-5.txt

```diff
@@ -0,0 +1,2 @@
+Requirement changes
+   * Drop support for MSVC 2013, 2015 and Arm Compiler 5.
```

Comment:
We should probably say `Visual Studio 2013 and 2015, and Arm Compiler 5`, rather than use the abbreviation `MSVC`. I'd also use put the comma in a different place this list, since it's (VS 2013 and 2015), and (AC5)

```suggestion
   * Drop support for Visual Studio 2013 and 2015, and Arm Compiler 5.
```